java/arcs/core/storage/DirectStore.kt

```diff
@@ -415,7 +409,17 @@ class DirectStore /* internal */ constructor(
          */
         private const val MAX_UPDATE_SPINS = 1000
 
+        suspend fun <Data : CrdtData, Op : CrdtOperation, T>create(options: StoreOptions<Data, Op, T>, type: CrdtModelType<Data, Op, T>): DirectStore<Data, Op, T> {
```

Comment:
This line needs formatting. Space before `create`, and probably put the args on separate lines. Probably this:

```
suspend fun <Data : CrdtData, Op : CrdtOperation, T> create(
    options: StoreOptiona<Data, Op, T>,
    type: CrdtModelType<Data, Op, T>
): DirectStore<Data, Op, T> {
```